docs/uglocalpv-hostpath.md

```diff
@@ -144,6 +152,8 @@ The default Storage Class is called `openebs-hostpath` and its `BasePath` is con
            value: hostpath
          - name: BasePath
            value: /var/local-hostpath
+         - name: NodeAffinityLabel
+           value: "openebs.io/node-affinity-value"
```

Comment:
`openebs.io/node-affinity-value` => `openebs.io/custom-node-unique-id`

Also, mention that the NodeAffinityLabel should have a unique value for each of the node in the cluster.